src/Monocle/Search/Query.hs

```diff
@@ -222,6 +245,9 @@ parseRelativeDateValue now txt = relTimeToUTCTime <$> readMaybe (from txt)
               Hour -> hour
               Day -> day
               Week -> day * 7
+              -- TODO Use calender aware computation
```

Comment:
How does it work for elasticsearch, is there a feature to adapt the bucket size to match the right time range? Or should we use average value, e.g. 30.4 day per month, and 365.25 per year.

Perhaps we should pad the query, for example a monthly interval range could be clamped to start at the beginning of the month.

Reply:
We use calendar_interval for Elasticsearch histogram. Elasticsearch interval start at the beginning of the interval period (from the elastic doc: "For example, if the interval is a calendar day, 2020-01-03T07:00:01Z is rounded to 2020-01-03T00:00:00Z."). [1]

Also we use "extended_bound" with min and max parameter which is the start and end date of the range. This make Elasticsearch to stick to the period and created the maximum number of buckets possible according the chosen interval period. [2]

[1]: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-datehistogram-aggregation.html#search-aggregations-bucket-datehistogram-aggregation
[2]: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-histogram-aggregation.html#search-aggregations-bucket-histogram-aggregation-hard-bounds